demos/orderly/src/depends/orderly.R

```diff
@@ -1,2 +1,3 @@
-orderly2::orderly_dependency("explicit", "latest", c(graph.png = "mygraph.png"))
+orderly2::orderly_dependency("explicit", "20240729-154639-25b955eb", c(graph.png = "mygraph.png"))
+orderly2::orderly_dependency("custom_metadata", "latest", "data.csv")
```

Comment:
It's good that we're covering using both 'latest' and specific ids here. For more coverage in manual testing, can we add a dependency to a packet that is not the latest in its group?

Reply:
added another pakcet to explicit.. so this now points to older dependency